### PR TITLE
TypeError: this.element.getAttribute

### LINKED ISSUE #3018: TypeError: this.element.getAttribute is not a function
### Describe the bug

TypeError: this.element.getAttribute is not a function showing up in js console

### To Reproduce

1. Open the patient finder gui

image

Notably, there is no impact on function. Everything works fine. The error message just keeps repeating in the console.

### Client configuration

- Browser: Firefox 72.0.2
- Operating system: ubuntu 16.04

### PR DESCRIPTION
**Pull request by im-Amitto**
Fixes #3018 

In #2995 function was modified to return value instead of echo but as it's used at other places where the echo was the expected value it was causing the problem so added a flag to the method.

### CODE REVIEW

**Comment by bradymiller** [comment]
nice! (I should of known better since I placed that feature to directly call setupAssets recently; not enough caffeine :) )

File: src/Core/Header.php [hunk 1]
```diff
@@ -94,7 +94,7 @@ public static function setupHeader($assets = [], $echoOutput = true)
      * @param boolean $headerMode - if true, then include autoloaded assets
      *                              if false, then do not include autoloaded assets
```
Comment (bradymiller) [inline src/Core/Header.php]: place the comments to the new parameter here; basically just copy/paste what is in above setupHeader comments

Comment (im-Amitto) [reply src/Core/Header.php]: How can I forgot about the documentation, I guess I need some sleep(. ❛ ᴗ ❛.).

**Comment by bradymiller** [comment]
hi @im-Amitto , Code looks good and bringing into codebase. thanks! -brady

**Pull request merged**